README.md

```diff
@@ -160,6 +160,7 @@ There is no free version of local Cloudant to download, so to develop or test wi
     - Click on the Icon, it will direct you to the database page. Check "Service Credentials" on the left to see your credentials.
     - Check "Manage" then click on button "LAUNCH" to see your Cloudant dashboard
 
+For cloudant on both DBaaS and Bluemix, to get access to the cloudant dashboard, you can sign in  https://cloudant.com/sign-in/ with your username and password.
```

Comment:
What username and password is this? Do you mean IBM ID, same as was used to log in to Bluemix?